app/decorators/case/base_decorator.rb

```diff
@@ -103,7 +103,11 @@ def status
     if current_state != 'closed'
       translation_for_case(object, "state", object.current_state)
     else
-      I18n.t("state.#{current_state}_status")
+      if object.ico?
+        "Closed - #{ico_decision.downcase} by ICO"
```

Comment:
It doesn't look like our `translation_for_case` method allows us to pass variables to the locales file. Looking at https://intranet.justice.gov.uk/guidance/security/building-security-pass/, I might have misunderstood how this works but it looks like options is not being passed to `I18n.t`

Reply:
We should fix the `translate_for_case` used here, since the one in lib does support it (I believe!).